src/actions/account.js

```diff
@@ -95,8 +95,8 @@ const wallet = new Wallet()
 
 export const redirectToApp = () => (dispatch, getState) => {
    const state = getState()
-   const nextUrl = (state.account.url && (state.account.url.success_url || state.account.url.public_key)) ? `/login/?${stringify(state.account.url)}` : '/'
```

Comment:
why /login got removed from path? I'm finding hard to follow what's going on.

Reply:
Generalized this function to redirect to any route, rather than just 'login'. 

The user's initial target url is stored in ```url.redirect_url``` when they're redirected to ```CreateAccount```. 

Should probably rename it from ```redirectToApp``` to something like ```redirectToRoute``` since it's not necessarily an app login.